src/model/guild/partial_guild.rs

```diff
@@ -450,4 +450,16 @@ impl PartialGuild {
     /// [Manage Webhooks]: permissions/constant.MANAGE_WEBHOOKS.html
     #[inline]
     pub fn webhooks(&self) -> Result<Vec<Webhook>> { self.id.webhooks() }
+
+    /// Obtain a reference to a role by its name.
+    ///
+    /// **Note**: If two or more roles have the same name, obtained reference will be one of them.
```

Comment:
I prefer functions to have examples where possible, could we make one here? Refer [here](https://github.com/zeyla/serenity/blob/master/src/model/channel/guild_channel.rs#L420-L516) for a good example of this.